frontend/src/components/EntityBulkAdd.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div v-for="num in entityFormComponentsCountRange" :key="`ef-${num}`">
+    <EntityForm :ref="`entity-form-${num}`" :schema="schema"
+                :attributes="attributes" :batch-mode="true" @save-all="saveAll"/>
+  </div>
+  <div class="container mt-2">
+    <button class="btn btn-outline-secondary w-100" @click="addEntityForm">
+      <i class='eos-icons'>add_circle</i>
+      Add more
+    </button>
+  </div>
+</template>
+
+<script>
+import EntityForm from "@/components/inputs/EntityForm.vue";
+
+export default {
+  name: "EntityBulkAdd",
+  components: {EntityForm},
+  props: {
+    schema: {
+      type: Object,
+      required: true
+    },
+    attributes: {
+      type: Object,
+      required: true,
+    }
+  },
+  data() {
+    return {
+      entityFormComponentCount: 1,
```

Comment:
This is not very nice, it's a shame that entity form isn't implemented as a v-model. It'd be very easy to hold a list of entities inside this components and `<EntityForm v-for="entity in entities" v-model="entity" />` and having a single "save all" button that could easily manage the bulk save from here. I'd say this would be the way to go, solving most of my other suggestions, but it's a probably a lot of work to refactor :disappointed:

Reply:
I agree that this would solve most of the suggestions.. I guess we can do the refactoring while switching to composition API? Or even sooner? IDK, I will think about it

Reply:
I guess there's no need to rewrite the entire component to composition API, only implementing the v-model in it, and fix the component usages throughout the repo (shouldn't be that many). I wouldn't say it's not a huge refactoring deserving its own issue/effort and I'd personally be fine updating this one while working on this issue (since it'd enable me to implement it properly much faster). In addition the issue is a 7, so it should be ok to spend some more time on it. But I can't speak for the others and it's ultimately up to you :slightly_smiling_face: